src/Texture2D.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio
+** File description:
+** Texture2D
+*/
+
+
+#include <raylib.h>
+#include "Texture2D.hpp"
+
+namespace indie {
+
+Texture::Texture(const std::string &fileName)
+{
+    _texture = std::make_unique<Texture2D>(::LoadTexture(fileName.c_str()));
```

Comment:
Why not use Texture::LoadTexture instead ?

Reply:
Because Texture::LoadTexture uses the one of the raylib anyway

Reply:
Sure but if you need to modify the encapsulation later you''ll need to do it two times, you are re writing the code instead of calling you're propper function.